data/json/scores/scores.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "id": "score_kills",
+    "type": "score",
+    "description": "Monsters killed: %s",
```

Comment:
Yeah, it felt a little redundant for a bunch of these. I'll wait for #39686 to land and then update.

I made the scores I added (and the existing ones) more concise than they were before, e.g. `Monsters killed: %s` rather than `Number of monsters killed: %s` because the original felt too long, and the only different between the score and the stat was just the leading capital, which I can easily update the stat to have and drop the score description.